### DISCUSSION TITLE
M600 on ender printers?

### DISCUSSION THREAD

**Discussion by Pgmiranda1** [supermerill/SuperSlicer#1973] [Community contribution]
Hi!
How to use M600 on ender printers?
Thenks!

**Comment by neophyl** [comment]
I don't think the normal Ender firmware supports M600.

You will have to find what gcode the Ender uses for colour change and then edit the the custom gcode section in Printer Settings>Custom Gcode>Colour Change Gcode to replace the M600 with whatever it does use.

**Reply by supermerill** [reply]
it support m600, even on the old ender3 (i have it).
I even added a ender3-mmu to be able to do manual mmu with multiple colors on the same layer (but i advise you to activate the purge tower, as it helps a lot for having a nice start).
